bbl/src/cpp_context.cpp

```diff
@@ -2,7 +2,6 @@
 #include "astutil.hpp"
 #include "bbl-detail.h"
 #include "bblfmt.hpp"
-#include "process.hpp"
```

Comment:
Let me know if I'm missing something related to `process.hpp`.

Is this file perhaps sitting uncommitted in your tree? Things seem to compile fine without it so I removed this `#include` but I'm on Linux so my build environment might be different.